pinc/forum_interface_phpbb3.inc

```diff
@@ -61,7 +61,7 @@ function phpbb_lang($locale = false)
         $locale = get_desired_language();
     }
 
-    if (PHPBB_VERSION == 3) {
+    if (PHPBB_VERSION == 3) { /** @phpstan-ignore-line */
```

Comment:
For this file we should just rip out this check. Probably in a separate PR as it'll be space-noisy.